cmdb/utils/wraps.py

```diff
@@ -18,9 +18,13 @@
 Different wrapper functions for interface module
 """
 import logging
+import warnings
+import inspect
+
+import cmdb
 from functools import wraps
 
-from flask import request, abort
+string_types = (type(b''), type(u''))
```

Comment:
What advantage does this provide? Can't you just do isinstance(u"foo", str)?

Reply:
Bytes produce an instance of the bytes type instead of the str type. They may only contain ASCII characters. Bytes are the actual data. Strings are an abstraction. The combination between type(b'') and type(u'') can thus catch more special characters.